source-salesforce/README.md

```diff
@@ -0,0 +1,132 @@
+# Salesforce Source
```

Comment:
This readme can be removed, as it's relevant only for Airbyte.

Better to not have it, then to have one that's wrong.

Reply:
Removed in [ae13497](https://github.com/estuary/connectors/pull/1355/commits/ae134972e57190b963389f75daf5e7d6a1bf6d92)

Reply:
Removed in [a1c34e9](https://github.com/estuary/connectors/pull/1355/commits/a1c34e924c00ab8a950e247e7bff669e8fcd0abd)